OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Toggle Tangible Permission

# Description
This PR enables the user with the Edit Time Entry Toggle Tangible permission to be able to toggle the checkbox of any user's tangible on time entry.

## Related PRS (if any):
This frontend PR is related to the [#713](https://github.com/OneCommunityGlobal/HGNRest/pull/713) backend PR.

…

## Main changes explained:
- Added a new permission label for Edit Time Entry Toggle Tangible permission.
- Added conditional logic so that the user with the permission can check or uncheck the tangible on time entry.

## How to test (please watch the video):
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log in with an owner account
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without **Toggle Tangible Time Others** permission under Timelog Management, Edit Time Entries -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to check and uncheck the box next to Tangible: on another user's dashboard for time log entries.

## Screenshots or videos of changes:



https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/9633a908-3871-4ab6-8584-437650899406



## Note:
**Unmute the video**
